### ISSUE TITLE
New try on moving AI to headlessclient

### ISSUE THREAD

**Issue by LouDnl** [LouDnl/ArmA3_Wasteland.Altis#26]
Move createUnit to HC. Create function for createUnit which checks if HC is present.

**Comment by hobart666** [comment]
LouD I miss ya help man. this would be epic. But please dont kill this git like Cre4mpie did his wasteland one. I'm still pounding my head on the desk at times but getting alot more certain things I cant get at all without agent ransack & the commits. but at least those are there. I do not know what transpired that pushed you away from the wasteland community aside from what you told me but man you're greatly missed. So much in your mission should be vanilla. Just sayin'.  Are the commits deep in here for ref? I know the one you have and vanilla is vastly different in end-user functionality. I actually just want to use this mission on my second server but im having issues so i may go with stratis vanilla and start fresh.
Anyhoo, peace pal.

**Comment by LouDnl** [comment]
I just needed a break. Sold my own server so won't be hosting anymore.
Kinda done with shitty arma updates that break stuff.
What do you need help with?

Verstuurd vanaf mijn iPhone

Op 17 jan. 2016 om 22:23 heeft hobart666 <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:

LouD I miss ya help man. this would be epic. But please dont kill this git like Cre4mpie did his wasteland one. I'm still pounding my head on the desk at times but getting alot more certain things I cant get at all without agent ransack & the commits. but at least those are there. I do not know what transpired that pushed you away from the wasteland community aside from what you told me but man you're greatly missed. So much in your mission should be vanilla. Just sayin'. Are the commits deep in here for ref? I know the one you have and vanilla is vastly different in end-user functionality. I actually just want to use this mission on my second server but im having issues so i may go with stratis vanilla and start fresh.
Anyhoo, peace pal.

##

**Comment by hobart666** [comment]
well two things, Im getting errors from persistence in my rpt after trying to add supporters to a vanilla stratis (I have the supporters working well in my altis mission) and I was wanting to use the last LouD branch of your wasteland altis mission but can only run it with iniDB saving it gives me rpt errors as well in the db. I am more motivated in getting stratis working with the loadouts I just don't know what Im missing. This is the error I get: Edit; I figured it out it was the order it was reading the _data from extdb/loadaccount.sqf

**Comment by hobart666** [comment]
I just remembered you inviting me to your stratis mission months ago and noting how smooth it was, and just realized you have a repo for it!! Does it have supporter loadouts?  Ahh f- it im excited im gonna dl it and check it out, would the fnc_param stuff need to be fixed on it? I'm going back into my hole after this!

**Comment by hobart666** [comment]
Are there any external files needed for your stratis mission?

**Comment by LouDnl** [comment]
Check my other repo with cfg files

Verstuurd vanaf mijn iPhone

Op 18 jan. 2016 om 06:52 heeft hobart666 <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:

Are there any external files needed for your stratis mission?

##

**Comment by LouDnl** [comment]
My missions are not compatible with latest Arma updates. I haven't updated to latest AgentRev commits.

**Comment by hobart666** [comment]
Ok I'll comb over them and see what I can do I had to do that to my mission
when he messaged me the oops about turrets last month.

**Comment by LouDnl** [comment]
Just get all latest commits from Revs dev repo.

Verstuurd vanaf mijn iPhone

Op 18 jan. 2016 om 19:37 heeft hobart666 <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:

Ok I'll comb over them and see what I can do I had to do that to my mission
when he messaged me the oops about turrets last month.
On Jan 18, 2016 5:48 AM, "LouDnl" <notifications@github.com<mailto:notifications@github.com>> wrote:

> My missions are not compatible with latest Arma updates. I haven't updated
> to latest AgentRev commits.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/LouDnl/ArmA3_Wasteland.Altis/issues/26#issuecomment-172495353
> .

##

**Comment by hobart666** [comment]
Yep, I've done that the other day to mine taking care of that tomorrow,
anything I need to know about these db files? are they extdb2? I glanced
but haven't started yet really on that. I'm going to start with a fresh db
using rev's latest release files extdb and just add those tables you have
in the one sql in the repo.

Joshua Lee
